jobs/cluster/upgrade/Jenkinsfile

```diff
@@ -85,7 +85,13 @@ node('openshift-build-1') {
             }
 
             stage( "smoketest" ) {
-                // run smoketest job here, try{}catch to determine if it failed and  include details of job in success email
```

Comment:
I did not include another try/catch specifically for the smoke test here - there's a surrounding try/catch block already.

So we could just set `propagate: true` if we wanted to fail the upgrade job on smoke test failure. But instead we probably want to check the result and throw a custom error...

About details of the smoke test itself: I believe that a URL pointing to the specific smoke test job run is enough here.

Reply:
The smoketest should be informational. The upgrade has already happened by this point, so we want to finish the pipeline. However, the "success" email we sent out should include a highly visible warning that smoketests have failed along with a link to the failing smoketest job's output. This is why the original smoketest was counting relatively minor infractions like nodes being unschedulable. 

So, if result != SUCCESS, I suggest just setting a flag that can be interpreted by mail_success. mail_success can then tailor the output to include the warning and the URL.